store/mocks/DataStore.go

```diff
@@ -118,6 +118,34 @@ func (_m *DataStore) DeleteImage(ctx context.Context, id string) error {
 	return r0
 }
 
+// SetDeploymentDeviceCount provides a mock function with given fields: ctx, deploymentID, count
+func (_m *DataStore) SetDeploymentDeviceCount(ctx context.Context, deploymentID string, count int) error {
+	ret := _m.Called(ctx, deploymentID, count)
+
+	var r0 error
+	if rf, ok := ret.Get(1).(func(context.Context, string, int) error); ok {
```

Comment:
how can it work with `Get(1)` shouldn't it be `Get(0)`?

Reply:
https://github.com/mendersoftware/deployments/pull/431